components/browser/state/src/main/java/mozilla/components/browser/state/state/ContentState.kt

```diff
@@ -98,4 +100,5 @@ data class ContentState(
     val appIntent: AppIntentState? = null,
     val showToolbarAsExpanded: Boolean = false,
     val previewImageUrl: String? = null,
+    val slowScriptCallback: SlowScriptCallback? = null,
```

Comment:
Could we rename to be `slowScriptRequest`? This way will follow the same pattern as we have we other GeckoRequests, like [PromptRequest,WindowRequest and SearchRequest](https://github.com/mozilla-mobile/android-components/blob/2d5d1ef117551ebd69c5d52582b59cb59498e9f9/components/browser/state/src/main/java/mozilla/components/browser/state/state/ContentState.kt#L79-L82)?